packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/InteractionLayer/InteractionLayerContainer.spec.js

```diff
@@ -103,14 +103,67 @@ describe('Component > InteractionLayerContainer', function () {
       expect(wrapper.find(TranscribedLines)).to.have.lengthOf(1)
     })
 
+    it('should render TranscribedLines exclusively per frame', function () {
+      const activeTask = {
+        hidePreviousMarks: false,
+        hidingIndex: 0,
+        marks: [
+          {
+            id: '1',
+            frame: 0,
+            toolIndex: 0,
+            toolType: 'transcriptionLine',
+            x1: 100,
+            y1: 100,
+            x2: 400,
+            y2: 101
+          },
+          {
+            id: '2',
+            frame: 0,
+            toolIndex: 0,
+            toolType: 'transcriptionLine',
+            x1: 100,
+            y1: 200,
+            x2: 400,
+            y2: 201
+          },
+          {
+            id: '3',
+            frame: 1,
+            toolIndex: 0,
+            toolType: 'transcriptionLine',
+            x1: 100,
+            y1: 100,
+            x2: 400,
+            y2: 101
+          }
+        ]
+      }
+      const activeTranscriptionTask = Object.assign(activeTask, transcriptionTask)
+
+      const wrapper = shallow(
+        <InteractionLayerContainer.wrappedComponent
+          height={height}
+          width={width}
+          frame={1}
+          activeInteractionTask={activeTranscriptionTask}
+          workflow={{ usesTranscriptionTask: true }}
+        />
+      )
+
+      const { marks } = wrapper.find(InteractionLayer).props()
+      expect(marks).to.have.lengthOf(1)
+    })
```

Comment:
Could you add to this test by switching frames and then testing that there are two marks?